### DISCUSSION TITLE
How to define Manufacturer and model to be visible in Home Assistant?

### DISCUSSION THREAD

**Discussion by zarss** [sle118/squeezelite-esp32#388] [Q&A]
Hi. I have added new key with field manufacturer to the NVS, but HA doesn't read it. Any idea please?

**Comment by sle118** [comment]
Perhaps you should provide a bit of background on what you are trying to achieve

**Reply by zarss** [reply]
![image](https://github.com/sle118/squeezelite-esp32/assets/33965048/3004504f-664e-4e99-be29-0a116c19acc5)
